dependencies/jasper/pom.xml

```diff
@@ -283,7 +283,7 @@
     <dependency>
       <groupId>com.lowagie</groupId>
       <artifactId>itext</artifactId>
-      <version>2.1.7.js4</version>
+      <version>4.2.1</version>
```

Comment:
Just curious, why are we updating the `com.lowagie. itext` version?

Reply:
as we have excluded com.lowagie from jasperreports .for info 
https://stackoverflow.com/questions/67579217/maven-build-failing-to-download-com-lowagieitextjar2-1-7-js6-dependency